### ISSUE TITLE
typo in Shift Instructions overview

### ISSUE THREAD

**Issue by hope51607** [riscv/riscv-p-spec#8]
- In section 2.2.3 table 3, line No.11 No.12
    - KSLL16 operation should be
        ```
        rt.H[x] = SAT.Q15(ra.H[x] << rb[3:0]);

        (RV32: x=1..0, RV64: x=3..0)
        ```
    - KSLLI16 operation should be
        ```
        rt.H[x] = SAT.Q15(ra.H[x] << im4u);

        (RV32: x=1..0, RV64: x=3..0)
        ```

- In section 2.2.4 table 4, line No.4 No.8
    ```
    SRAI8.u rt, ra, im4u
    SRLI8.u rt, ra, im4u
    ```
    should be
    ```
    SRAI8.u rt, ra, im3u
    SRLI8.u rt, ra, im3u
    ```

**Comment by chuanhua** [comment]
Thanks for finding out the typos. I will fix them in the next release.

**Comment by chuanhua** [comment]
Fixed in v0.9.1 release. (976dbe7)
